README.md

```diff
@@ -160,7 +160,7 @@ Resources:
       ClientIdList: 
         - sts.amazonaws.com
       ThumbprintList:
-        - a031c46782e6e6c662c2c87c76da9aa62ccabd8e
+        - 15e29108718111e59b3dad31954647e3c344a231
```

Comment:
```suggestion
        - 6938fd4d98bab03faadb97b34396831e3780aea1
```

Reply:
Please see the guidance at https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc_verify-thumbprint.html on obtaining the thumbprint.

Reply:
This works too:

```sh
openssl s_client -connect token.actions.githubusercontent.com:443 < /dev/null 2>/dev/null | openssl x509 -fingerprint -noout -in /dev/stdin
```

Reply:
I obtained the thumbprint as https://github.com/aws-actions/configure-aws-credentials/issues/357#issuecomment-1011642085. It is `6938fd4d98bab03faadb97b34396831e3780aea1`.